src/daft-dsl/src/lit.rs

```diff
@@ -212,6 +212,36 @@ impl LiteralValue {
         };
         result
     }
+
+    pub fn to_sql(&self) -> Option<String> {
```

Comment:
no need to implement but you could have this be `display_sql` and take in a formatter as well!